cms/djangoapps/contentstore/utils.py

```diff
@@ -1590,21 +1590,25 @@ def get_library_context(request, request_is_json=False):
         user_can_create_library,
     )
 
+    is_migrated: bool | None  # None means: do not filter on is_migrated
```

Comment:
all this new parameter handling is functionally equivalent for all the 200 cases. it's just being a bit more careful with parsing, so that some things that would have been 500s are now 400s with validation messages.